projects/libs/flex-layout/package.json

```diff
@@ -29,7 +29,7 @@
     "tslib": "^2.3.0"
   },
   "peerDependencies": {
-    "@angular/cdk": "^14.0.0",
+    "@angular/cdk": "^13.0.0 || ^14.0.0",
```

Comment:
@SamuelMarks it should go the other way. ^14.0.0 || ^15.0.0 that way when Angular 15 comes out, people with Flex-Layout 14 won't have trouble upgrading to angular 15, while this team works on updating flex-layout to 15

Reply:
Ah, ok just did `fastmod -F '^13.0.0 || ^14.0.0' '^14.0.0 || ^15.0.0'`